src/CLI_Full/CLI_Full.csproj

```diff
@@ -27,6 +27,10 @@
     <Compile Include="..\CLI\*.cs" Link="*.cs (From CLI)" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
CLI_Full "piggybacks" on the code from the CLI project. Since the resources are in a separate folder (CLI\Resources), we explicitly include that folder. I wasn't able to get glob patterns to work here ☹️

Reply:
Oh didn't think about those files. Yes in that case, I think two simple lines are way better than a single complex one.